.devcontainer/welcome-message.txt

```diff
@@ -0,0 +1,7 @@
+👋  Welcome to Flask in Codespaces!
+
+🛠️  Your environment is setup with all the required dependencies and your fork has been added as a remote.
+
+🚀  To start coding:
+    - wait for the "postCreateCommand" to finish setting things up
+    - run `git checkout -b your-branch-name origin/{main-or-latest-release}` to start working on a new feature or bug fix.
```

Comment:
This message will only show at initial codespace creation or if a codespace is rebuilt.